docs/src/providers/WidgetProvider.tsx

```diff
@@ -58,43 +56,47 @@ export function WidgetProvider({ children }: { children: React.ReactNode }) {
     getEvents,
   } = useBoundWidgetStore();
 
-  const [rightPanelWidgets, setRightPanelWidgets] = useState<Widget[]>();
-  const [sendableWidgets, setSendableWidgets] = useState<Widget[]>();
-
-  useEffect(() => {
-    const initialWidget: Widget = {
-      id: faker.string.uuid(),
-      name: "React Example",
-      url: "http://localhost:3001",
-      carrierIds: [1],
-      scopes: [
-        {
-          location: "conversations",
-          positions: ["right_panel"],
-        },
-      ],
-      dealers: {
-        allDealers: true,
-      },
-      type: "iframe",
-      priority: 99,
-    };
-    setWidgets([initialWidget]);
-    setRightPanelWidgets([initialWidget]);
-  }, [setWidgets]);
+  const [rightPanelWidgets, setRightPanelWidgets] = useState<Widget[]>([]);
+  const [sendableWidgets, setSendableWidgets] = useState<Widget[]>([]);
+
+  // useEffect(() => {
```

Comment:
Should this be removed?

Reply:
I left it for now as reference for if we're able to get an example widget set up